Binary files protobuf-2.6.1 2/.DS_Store and protobuf-2.6.1 3/.DS_Store differ
Binary files protobuf-2.6.1 2/gtest/.DS_Store and protobuf-2.6.1 3/gtest/.DS_Store differ
diff -rupN protobuf-2.6.1 2/gtest/build-aux/ltmain.sh protobuf-2.6.1 3/gtest/build-aux/ltmain.sh
--- protobuf-2.6.1 2/gtest/build-aux/ltmain.sh	2014-10-22 13:10:23.000000000 -0700
+++ protobuf-2.6.1 3/gtest/build-aux/ltmain.sh	2017-09-23 12:14:16.000000000 -0700
@@ -8721,16 +8721,16 @@ EOF
 
       case $host in
       *-*-darwin*)
-	# Don't allow lazy linking, it breaks C++ global constructors
-	# But is supposedly fixed on 10.4 or later (yay!).
-	if test "$tagname" = CXX ; then
-	  case ${MACOSX_DEPLOYMENT_TARGET-10.0} in
-	    10.[0123])
-	      func_append compile_command " ${wl}-bind_at_load"
-	      func_append finalize_command " ${wl}-bind_at_load"
-	    ;;
-	  esac
-	fi
+# 	# Don't allow lazy linking, it breaks C++ global constructors
+# 	# But is supposedly fixed on 10.4 or later (yay!).
+# 	if test "$tagname" = CXX ; then
+# 	  case ${MACOSX_DEPLOYMENT_TARGET-10.0} in
+# 	    10.[0123])
+# 	      func_append compile_command " ${wl}-bind_at_load"
+# 	      func_append finalize_command " ${wl}-bind_at_load"
+# 	    ;;
+# 	  esac
+# 	fi
 	# Time to change all our "foo.ltframework" stuff back to "-framework foo"
 	compile_deplibs=`$ECHO " $compile_deplibs" | $SED 's% \([^ $]*\).ltframework% -framework \1%g'`
 	finalize_deplibs=`$ECHO " $finalize_deplibs" | $SED 's% \([^ $]*\).ltframework% -framework \1%g'`
diff -rupN protobuf-2.6.1 2/ltmain.sh protobuf-2.6.1 3/ltmain.sh
--- protobuf-2.6.1 2/ltmain.sh	2014-10-22 13:10:25.000000000 -0700
+++ protobuf-2.6.1 3/ltmain.sh	2017-09-23 12:13:44.000000000 -0700
@@ -8721,16 +8721,16 @@ EOF
 
       case $host in
       *-*-darwin*)
-	# Don't allow lazy linking, it breaks C++ global constructors
-	# But is supposedly fixed on 10.4 or later (yay!).
-	if test "$tagname" = CXX ; then
-	  case ${MACOSX_DEPLOYMENT_TARGET-10.0} in
-	    10.[0123])
-	      func_append compile_command " ${wl}-bind_at_load"
-	      func_append finalize_command " ${wl}-bind_at_load"
-	    ;;
-	  esac
-	fi
+# 	# Don't allow lazy linking, it breaks C++ global constructors
+# 	# But is supposedly fixed on 10.4 or later (yay!).
+# 	if test "$tagname" = CXX ; then
+# 	  case ${MACOSX_DEPLOYMENT_TARGET-10.0} in
+# 	    10.[0123])
+# 	      func_append compile_command " ${wl}-bind_at_load"
+# 	      func_append finalize_command " ${wl}-bind_at_load"
+# 	    ;;
+# 	  esac
+# 	fi
 	# Time to change all our "foo.ltframework" stuff back to "-framework foo"
 	compile_deplibs=`$ECHO " $compile_deplibs" | $SED 's% \([^ $]*\).ltframework% -framework \1%g'`
 	finalize_deplibs=`$ECHO " $finalize_deplibs" | $SED 's% \([^ $]*\).ltframework% -framework \1%g'`
